3-terrarium/2-intro-to-css/README.md

```diff
@@ -35,6 +36,7 @@ Cascading Style Sheets incorporate the idea that the styles 'cascade' such that
 
 ### Task
 
+This is not a really popular way you should do it.<br>
 Add the inline style "color: red" to your `<h1>` tag:
 
```

Comment:
Let's not add this comment, as it's a judgement call I don't want to explain in this section